assets/package.json

```diff
@@ -78,12 +80,13 @@
     "autotrack": "^2.4.1",
     "d3": "^5.5.0",
     "fast-memoize": "^2.5.1",
+    "list.js": "^1.5.0",
     "lodash": "^4.17.10",
     "raf-throttle": "^2.0.3",
     "redux": "^4.0.1",
     "redux-thunk": "^2.3.0",
     "reselect": "^3.0.1",
     "resize-observer-polyfill": "^1.5.0",
-    "uswds": "^2.0.0-alpha"
+    "uswds": "2.0.0-alpha"
```

Comment:
I had to pin this to this version. The latest 2.0.0-beta.3 would not build. Will open a ticket to investigate why?